subgraph/schema.graphql

```diff
@@ -33,6 +33,8 @@ type Lock @entity {
   deployer: Bytes!
   "Total number of receipts of lock"
   numberOfReceipts: BigInt!
+  "Referrers fees set per custom address"
+  referrerFees: [ReferrerFee!]! @derivedFrom(field: "lock")
```

Comment:
We've added `referrerFees` to the Lock entity to track the referrer fees set for each address.

**Improvement Tip** _(if it doesn't align with the product stack)_: Keep using `referrerFees` in the Lock but switch to a format like `${referrer}-${fee}`, for instance `0x123...-200`. This array of strings can be processed with `.split("-")` in the browser to extract the address and fee percentage.

Reply:
@julien51 Let me know if this still suits you 👍

Reply:
It does!